Buildings/Resources/src/fmi-library/CMakeLists.txt

```
@@ -7,7 +7,7 @@ cmake_policy(SET CMP0048 NEW)
 cmake_minimum_required(VERSION 3.8)
 
 project( fmi-library
-  VERSION 2.2
+  VERSION 3.0
```

```suggestion
  VERSION 3.0.1
```
Hi @mwetter would it be possible to bump this to 3.0.1? We just released 3.0.1 a few minutes ago and it fixes a couple the CVE's we had seen in our third-party components. Same suggestion for below.

---

Update: We will release FMIL 3.0.2 either today or tomorrow. We found a regression in 3.0.1 where unzipping is printing to stdout. Please keep this PR open until I have updated the suggestions again.

---

Update: Bumped from 3.0.1 to 3.0.2. Fixes an issue introduced in 3.0.1.